release_notes/371-release-notes.adoc

```diff
@@ -0,0 +1,225 @@
+:_content-type: ASSEMBLY
+[id="release-notes-371"]
+= Red Hat Advanced Cluster Security for Kubernetes 3.71
+include::modules/common-attributes.adoc[]
+:context: release-notes-371
+
+toc::[]
+
+*Release date:* 25 July 2022
+
+{product-title} ({product-title-short}) is an enterprise-ready, Kubernetes-native container security solution that protects your vital applications across build, deploy, and runtime. It deploys in your infrastructure and integrates with your DevOps tooling and workflows to deliver better security and compliance and to enable DevOps and InfoSec teams to operationalize security.
+
+[id="about-this-release-371"]
+
+{product-title-short} 3.71 includes feature enhancements, bug fixes, scale improvements, and other changes.
+
+[id="new-features-371"]
+== New features
+
+[id="dashboard-updates"]
+=== Dashboard updates
+
```

Comment:
"Select" should only be used when referring to a mouse click: 

Use with caution select   verb
Use "select" to indicate a mouse action that applies to both mobile and non-mobile devices when a term such as "click" might create confusion. 

Wondering if you could remove it for ". . . for all clusters and namespaces or select individual clusters. . ."

Reply:
So it is a checkbox that you have to mark to choose either "all clusters" or pick specific ones, so I feel like "select" or "choose" or "check the box" is needed here. But I'm confused b/c IBM Style Guide says that select is OK and uses it in its examples: https://www.ibm.com/docs/en/ibm-style?topic=elements-ui#verbs-to-use  Is this one of those cases where RH and IBM are different?

Reply:
@kcarmichael08 I see the places where the d's should be lower (id field) and upper (New RHACS Dashboard).